hack/test-e2e.sh

```diff
@@ -72,10 +72,9 @@ else
     echo $KUBECONFIG
     oc project || :
     oc config current-context || :
-    oc describe pod elasticseach-operator-e2e-test -n ${TEST_NAMESPACE}
 
     # wait for the pod to be ready first...
-    oc wait pod/elasticsearch-operator-e2e-test -n ${TEST_NAMESPACE} --for condition=Ready || oc logs elasticsearch-operator-e2e-test
+    oc wait pod/elasticsearch-operator-e2e-test -n ${TEST_NAMESPACE} --for condition=Ready || oc describe pod elasticsearch-operator-e2e-test -n ${TEST_NAMESPACE}
```

Comment:
I think you need to wait longer.  The status says it is pulling the image from the ci repo - that may take a few minutes - I believe logging waits 5 minutes for this - the default `oc wait` timeout is 30s
```
      --timeout=30s: The length of time to wait before giving up.  Zero means
check once and don't wait, negative means wait for a week.
```
so maybe use `--timeout=300s`